CMake_VS2015.bat

```diff
@@ -1,9 +1,35 @@
 @ECHO OFF
 @echo:
 @echo:
-ECHO Generating Visual Studio Solution (64 bit)
+SET ARG=%1
+@echo:
 @echo:
+ECHO Generating Visual Studio Solution (64 bit)
 @echo:
+SET BUILDMODE="d3d9"
+IF DEFINED ARG (
+    IF "%ARG%"=="--opengl" (
+    	SET BUILDMODE="opengl"
+		ECHO OpenGL configuration selected.
+	)
+	IF "%ARG%"=="/opengl" (
+    	SET BUILDMODE="opengl"
+		ECHO OpenGL configuration selected.
+	)
+    IF "%ARG%"=="--d3d11" (
+    	SET BUILDMODE="d3d11"
+		ECHO Direct3D 11 configuration selected.
+	)
+    IF "%ARG%"=="/d3d11" (
+    	SET BUILDMODE="d3d11"
+		ECHO Direct3D 11 configuration selected.
+	)
+)
+IF %BUILDMODE%=="d3d9" (
+  ECHO Direct3D 9 configuration selected by default (Use /opengl or /d3d11 switches to specify OpenGL or DirectX 11)
+
```

Comment:
Makes sense - although I think the .bat and .sh could still be useful as a convenience to create the workspaces just by double-clicking in the file manager rather than having to open a command prompt and pass flags.